docs/concepts.rst

```diff
@@ -1,11 +1,86 @@
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+The Nonprofit Success Pack, or NPSP, is one of Salesforce.org's flagship products. The NPSP consists of six managed packages, with complex dependency relationships between them, alongside unpackaged metadata and setup automation that helps create a usable NPSP org.
+
+Salesforce.org delivers NPSP using CumulusCI and the Product Delivery Model because those tools are what make it possible for us to ship this large, complex, heterogeneous application efficiently to tens of thousands of customers — without a lengthy setup guide to be completed by the end user. CumulusCI automation seamlessly deploys all six managed packages in the right sequence, delivers the unpackaged metadata, like Record Types and Page Layouts, we supply to make the customer's life easier, and executes setup automation like populating NPSP relationship values. 
+
+CumulusCI runs this automation throughout our development lifecycle, starting from feature branches in the hands of our developers and culminating the delivery of new versions of the application to our users, including through MetaDeploy — which runs the very same automation we use internally to set up and configure a customer org.
+
+Throughout the CumulusCI documentation, we'll have the Product Delivery Model in mind. 
 
 Projects
 --------
 
-Tasks and flows
+When you work with CumulusCI, you do so inside a project. A project is a version control repository that contains Salesforce metadata as well as the CumulusCI automation that builds and releases the project. A project usually has a one-to-one relationship with a managed package, if building a package, or with an org implementation. A project may constitute the entirety of a product in the Product Delivery Model, or may be one of multiple projects making up a product.
+
+CumulusCI scopes many of its activities to the project, so you'll always run your CumulusCI commands inside your repository directory. Each project gets its own keychain, which we'll talk about in detail below. Each project has its own set of scratch orgs, which CumulusCI namespaces in the Salesforce DX keychain to prevent collisions. And each project is linked to a single GitHub repository, so you can always be confident of the scope of the actions you're taking.
+
+CumulusCI knows which project you are working on by the current working directory of your shell. Make sure to change directories inside your project before you run project-specific commands.
+
+Managing Orgs & Services with the Keychain
+------------------------------------------
+
+CumulusCI gives each project a keychain, and also offers a global keychain that's shared across projects. The keychain's role is to store access information for all of the orgs you're using with the project — both scratch orgs and persistent orgs — and the details of the services you've connected, such as a GitHub account or an instance of MetaDeploy.
+
+Services
+++++++++
```

Comment:
How about covering orgs first, since they are a concept that the reader is more likely to already have. Then services can be explained as "credentials/config for other external services that aren't orgs"